_src/static/sass/base/_base.scss

```diff
@@ -0,0 +1,92 @@
+/* ============================
+-------------------------------
+ 1. Basic Setup
+-------------------------------
+============================ */
+
+
+/* ============================
+1. Basic Setup
+---------------------------- */
+html {
+    height: 102%;
+}
+
+body {
+    font-size: $font-size;
+    min-width: $fg-min-width;
+    background: $gray-lightest;
```

Comment:
We have not defined any views yet, so I would leave this undefined and out for now.